### ISSUE TITLE
How to add devices manually?

### ISSUE THREAD

**Issue by Ndrinta** [home-assistant/home-assistant.github.io#16876]
### Feedback

I have upgraded to the new integration and I saw it populated a series of new entities (linked to ASUSWRT) which I guess are the devices currently connected to the network, but what about the devices offline at the moment? I couldn't find an option to add devices manually and, even though I know you can, I refuse to add offline devices brute forcing them into the "core.xxx" files, it makes little sense since the new integration aims is to simplify things.
Not all the devices I use in automation (or simply linked in Lovelace) are currently connected to the network but I still want to make this upgrade a 1 day job, not a recurrent thing I'll have to follow up in the upcoming weeks (waiting for all the devices to slowly show up to fix their friendly names and entities ID). Like most of us, I cozily have the old list of devices ('known_devices.yaml'.) readily available and I already know all the MAC addresses, I would like to incorporate the new ASUSWRT "list of devices" with the offline devices manually.
Furthermore, lots of automation based on these "not yet connected" devices will not work if I don't fix the entity ID earlier because the new names given by the integration the first time will see them will (most likely) differ from the ones I had in 'known_devices.yaml' (witch I now backed up and cleared).

It would have been nice to have an import option to bring in your 'known_devices.yaml' list and convert it into the new ASUSWRT standard (maybe with a way to manually handle conflicts).
I mean, it's cool what you've done but you made my life a little harder instead of easier this time.

### URL

https://www.home-assistant.io/integrations/asuswrt/

### Version

2021.3.X

**Comment by frenck** [comment]
> I couldn't find an option to add devices manually and, even though I know you can

You can't add them manually, they will show up once detected the first time.

All comments in this issue are towards the implementation details, feedback on the integration and contain feature requests. All of those, are not related to the documentation. Please join our community forums, as that is the place for this type of discussion.

Thanks 👍

**Comment by Ndrinta** [comment]
![image](https://user-images.githubusercontent.com/4704172/110837279-ef845880-8298-11eb-9d6c-793bf335cb2f.png)

I just don't see how this could be a good idea... I just spent my evening renaming every single entity (changing entity ID too) and then I opened the Device page and whaaaat.
I made a significant effort in the past in keeping everything tide and clean (and frenk I know you care, I saw your configuration.yaml) but I just give up here.
I just think that it was a mistake. A nice logo doesn't make up for the complete loss of usability.
